projects/admin/src/app/routes/route-tool.service.ts

```diff
@@ -196,13 +206,13 @@ export class RouteToolService {
    * Get route param by name
    *
    * @param name - string
+   * @param defaultValue: the default value to return if name is not found as query parameter
    * @return mixed - string | null
    */
-  getRouteQueryParam(name: string) {
+  getRouteQueryParam(name: string, defaultValue = null) {
```

Comment:
documentation is missing. The use of this should be generalized in the whole library. Why not in ng-core?

Reply:
add more docstring.
You 're right, maybe it could take place into an `utils` file into `ng-core` project.